llama_stack/providers/remote/inference/ollama/docker_compose.yaml

```diff
@@ -0,0 +1,55 @@
+services:
```

Comment:
Should this `docker_compose.yaml` be inside /providers?

Reply:
yeah killed it